content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -295,9 +265,9 @@ Once you have your stock `rippled` server running, you may want to consider runn
 
 `rippled` should connect to the XRP Ledger with the default configurations. However, you can change your settings by editing the `rippled.cfg` file. For recommendations about configuration settings, see [Capacity Planning](capacity-planning.html).
 
-Changes to the `[node_db]`, `[debug_logfile]`, or `[database_path]` sections may require you to give the `rippled` user and group ownership to your new configured path: ***TODO: I added [node_db] to this list - okay? When I try the following command, `chown -R rippled:rippled /var/lib/rippled/db/rocksdb`, I get `chown: rippled: illegal group name.` Is `rippled:rippled` just an example below -- or should there be a `rippled` group that is created by the build/install process?***
+Changes to the `[node_db]`, `[debug_logfile]`, or `[database_path]` sections may require you to give the `rippled` user and group ownership to your new configured path: ***TODO: I added [node_db] to this list - okay?***
 
-    $ chown -R rippled:rippled <configured path>
+    $ chown -R rippled:rippled <configured path> ***TODO: Is `rippled:rippled` just an example value? When I run this command on macOS, I get `chown: rippled: illegal group name.` When I run it on centos, I get chown: changing ownership of ... : Operation not permitted***
```

Comment:
This group doesn't exist on your Mac that's why you get this error. 
When you spin up aws machines to test centos/Ubuntu (I assume that's what you are doing) this user group is created by default (OPS has this configured) hence you don't get this error. 

But you have the right idea to give permissions to the user who is going to run rippled (maybe we don't need to give an example of how to change permissions? but just say we need to get the right permission to start rippled)

Reply:
Options:

1. Tell the user to run rippled as their own user. This is a reasonable balance of safe and easy—not as safe as running as a dedicated user like `rippled`, not as easy as running as root, but in between. Change the example command to use `$USER` (the environment variable for the current user, assuming that's also valid on macOS like it is on Linux).
2. Tell the user to create a `rippled` user for safety and provide steps for (a) how to create such a user, (b) how to give it the right permissions, and (c) how to run `rippled` as that user. This is fairly standard system admin-level stuff, which may or may not be a bit too much work for the intended audience here. We're already telling them not to run on macOS in production so taking production-level precautions is probably too much here.

I'd lean towards 1 since it's more expedient. There are third-party docs they can leverage if they want to re-create a production-like environment on macOS despite our recommendations, and we probably shouldn't provide recommendations for things we don't support anyway.